src/dodal/beamlines/i03.py

```diff
@@ -204,6 +207,21 @@ def fast_grid_scan(
     )
 
 
+def panda_fast_grid_scan(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> PandAFastGridScan:
+    """Get the i03 fast_grid_scan device, instantiate it if it hasn't already been.
```

Comment:
Should: The description here doesn't mention `panda`, I think some mention of how this is the one that should be used when triggering with the panda (rather than the FGS) would be good too